.circleci/scripts/binary_linux_test.sh

```diff
@@ -118,6 +118,12 @@ if [[ "$PACKAGE_TYPE" == libtorch ]]; then
   cd /tmp/libtorch
 fi
 
+if [[ "$GPU_ARCH_TYPE" == xpu ]]; then
+  # Workaround for __mkl_tmp_MOD unbound variable issue
```

Comment:
Can you please add a link to the issue here

Reply:
Added